lib/index.js

```diff
@@ -41,6 +41,21 @@ Intercom.ensure(function(msg){
   }
 });
 
+Intercom.prototype.initialize = function(){
+  this.userAgent = 'Segment.io/1.0.0';
+  // Intercom asked us to split some users on the old version to help with load
+  if (this.settings.isV1Enabled) {
+    this.identify = this.identifyV1;
+    this.track = this.trackV1;
+    this.group = this.groupV1;
+  } else {
+    this.identify = this.identifyV2;
```

Comment:
@thehydroimpulse this was more like us making quick copy and pastes so we can mitigate the SEV asap

Is using method names like this have any negative technical implications other than readability? I'd imagine once the SEV is situated, all this will change once Intercom and Segment figures out a long term solution

cc @tsholmes